test/lit.cfg

```diff
@@ -937,25 +937,33 @@ if run_vendor == 'apple':
     # The "freestanding" tests will link against the static libswiftCore.a and
     # cannot use any of Obj-C / Dispatch / Foundation.
     if "-freestanding" in config.variant_suffix:
-        config.target_runtime = "native"
-        config.available_features.remove('libdispatch')
-        config.available_features.remove('foundation')
-        config.available_features.remove('objc_interop')
+        if not config.swift_freestanding_is_darwin:
+            config.target_runtime = "native"
+        if(not config.swift_enable_dispatch):
```

Comment:
You should be able to drop the '('. And also a few lines below.

Reply:
This is addressed in dc0e991